flutter/lib/src/integrations/frames_tracking_integration.dart

```diff
@@ -0,0 +1,73 @@
+import '../../sentry_flutter.dart';
+import '../binding_wrapper.dart';
+import '../frames_tracking/sentry_delayed_frames_tracker.dart';
+import '../frames_tracking/span_frame_metrics_collector.dart';
+import '../native/sentry_native_binding.dart';
+
+class FramesTrackingIntegration implements Integration<SentryFlutterOptions> {
+  FramesTrackingIntegration(this._native);
+
+  final SentryNativeBinding _native;
+  SentryFlutterOptions? _options;
+  PerformanceCollector? _collector;
+  SentryWidgetsBindingMixin? _widgetsBinding;
+
+  @override
+  Future<void> call(Hub hub, SentryFlutterOptions options) async {
+    _options = options;
+
+    void abortInitWith({required String reason}) {
```

Comment:
Nit: we could rename the method and required parameter. It's only called within this method and would make the calls a bit more readable. Also it's only a logging helper, so the function name could reflect that.

```dart
abortInitWith(reason: "foo")
```

vs 

```dart
log("bar")
```

Reply:
it's only a few lines, I just logged it separately without a helper function